src/App.js

```diff
@@ -17,29 +21,95 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
-  // Setup state here...
+  const [store, setStore] = useState(initialStoreItems)
+  const [cart, setCart] = useState([])
 
+
+  // making a new array of a cart by adding items
+  const addNewItem = (item) => {
+    setCart([...cart, item])
+    console.log("hello".item)
+  }
+
+  // updating items
+  const onIncreaseClick = (target) => {
+    console.log("Hi", target)
+    const updatedQuantity = cart.map(item => item.id === target.id ? { ...item, quantity: item.quantity + 1 } : item)
+    setCart(updatedQuantity)
+  };
+
+  // removing items
+  const onDecreaseClick = (target) => {
+    if (target.quantity === 1) {
```

Comment:
This is a nice way to do this, only concern is 'onDecreaseClick' isn't descriptive enough for the function since it can do two different things (can decrease quantity, or remove the item if quantity '1'.
Another option is to just have it set quantity to 0, and then just not render cart items with quantity === 0.